docs/uploader/transloadit.mdx

```diff
@@ -26,12 +26,13 @@ Not sure which uploader is best for you? Read
 Transloadit’s strength is versatility. By doing video, audio, images, documents,
 and more, you only need one vendor for [all your file processing
 needs][transloadit-services]. The `@uppy/transloadit` plugin directly uploads to
-Transloadit so you only have to worry about creating a
-[Template][transloadit-concepts]. Transloadit accepts the files, processes
-according to the instructions in the Template, and stores the results in storage
-of your choosing, such as a self-owned S3 bucket. The Transloadit plugin uses
-[Tus](/docs/tus) under the hood so you don’t have to sacrifice reliable,
-resumable uploads.
+Transloadit so you don’t have to worry about setting up and maintaining your own
+servers, all you have to do is simply create a [Template][transloadit-concepts]
```

Comment:
Words like "simply" and "just do x" are my personal pet peeves. Other than that LGTM

```suggestion
servers, all you have to do is create a [Template][transloadit-concepts]
```